File: MTGuard/MTGuard/ViewController.m
//
//  ViewController.m
//  MTGuard
//
//  Created by moshuqi on 16/7/23.
//  Copyright © 2016年 msq. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^(){
        UIView *v = [UIView new];
        [self.view addSubview:v];
    });
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: MapPath/MapPath/ViewController.swift
//
//  ViewController.swift
//  MapPath
//
//  Created by moshuqi on 16/2/26.
//  Copyright © 2016年 msq. All rights reserved.
//

import UIKit

class ViewController: UIViewController, MAMapViewDelegate{
    
    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    @IBAction func run(sender : AnyObject) {
        let runningViewController = RunningViewController()
        presentViewController(runningViewController, animated: true, completion: nil)
    }
    
    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */

}
